Documentation/PropertyWrapper/Parameter.md

```diff
@@ -184,10 +184,10 @@ Exposing the `Component` on a web socket interface requires the client to send t
 
 ## Lightweight Parameters
 
-Some middleware types and protocols can expose parameters as lightweight parameters that can be part of a URI path such as query parameters found in the URI of RESTful and OpenAPI based interfaces. Apodini automatically exposes primative types as lightweight parameters that conform to [`LosslessStringConvertible`](https://developer.apple.com/documentation/swift/losslessstringconvertible). Complex types such as cunstom types conforming to Codable can not be exposed using lightweight parameters. A lightweight parameter is defined the same way a normal parameter can be defined. The developer using Apodini can force a specific behaviour by passing in an enum value into the property warpper initializer, e.g. `@Parameter(.lightweight)`. Possible values are:
-* `.automatic`: Apodini infers the type of parameter based on the type information of the wrapped propety. This is the default behaviour of an `@Parameter` property wrapper.
+Some middleware types and protocols can expose parameters as lightweight parameters that can be part of a URI path such as query parameters found in the URI of RESTful and OpenAPI based interfaces. Apodini automatically exposes primitive types as lightweight parameters that conform to [`LosslessStringConvertible`](https://developer.apple.com/documentation/swift/losslessstringconvertible). Complex types such as custom types conforming to `Codable` can not be exposed using lightweight parameters. A lightweight parameter is defined the same way a normal parameter can be defined. The developer using Apodini can force a specific behaviour by passing in an enum value into the property wrapper initializer, e.g. `@Parameter(.lightweight)`. Possible values are:
+* `.automatic`: Apodini infers the type of parameter based on the type information of the wrapped property. This is the default behaviour of an `@Parameter` property wrapper.
```

Comment:
I am not sure if this is the right place to introduce (or summarize) the possible options for `@Parameter` as we are in the section ##Lightweight Parameters. Maybe we should move this to the very beginning, as this is a more general paragraph and would be the practical implication of the fact, that "property wrapper can be used to express input in different ways". At least I found it somewhat irritating that the use of `.content` is explained here, as it is not a lightweight parameter.

Reply:
I agree with you and will move this section to the beginning of the document.